Language/SerialKillers_localization.en_us.conf

```diff
@@ -0,0 +1,10 @@
+StringTableRuntime {
+ Ids {
+  "SK_CAR_UAZ469_Police"
+  "SK_CAR_UAZ469_Police_Blue"
+ }
+ Texts {
+  "UAZ469 Police"
+  "UAZ469 Police Blue"
+ }
```

Comment:
I hate to be that guy, but it's not `Police`, but `Militia` UAZ. Not only for text (visible name), but for classname too.

Try simple replace in all files:
- `SK_CAR_UAZ469_Police` -> `SK_CAR_UAZ469_Militia`
- `SK_CAR_UAZ469_Police_Blue` -> `SK_CAR_UAZ469_Militia_Blue`

Hopefully it will work.

Reply:
Now I see there's `Police` all over the place. It would need replacing too.